programs/token-2022/src/instructions/extensions/memo_transfer/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod instructions;
```

Comment:
I don't think you need another `instructions` module. Remember that extensions are already inside an `instructions` module.